templates_jinja2/base.html

```diff
@@ -72,7 +72,7 @@
         <input type="hidden" name="cx" value="017956276086362105630:99ctqytcokm" />
         <input type="hidden" name="cof" value="FORID:11" />
         <input type="hidden" name="ie" value="UTF-8" />
-        <input type="text" class="form-control typeahead" placeholder="Search for teams and events" name="q"/>
+        <input type="text" class="form-control typeahead" placeholder="🔎 Search teams and events" name="q"/>
```

Comment:
Upon further review, the ASCII character doesn't always render on all platforms,  and to use a glyphicon in there is slightly messy. I'll just back them out.